format-common/src/main/kotlin/app/passwordstore/data/passfile/PasswordEntry.kt

```diff
@@ -217,5 +217,7 @@ constructor(
         "secret:",
         "pass:",
       )
+    private const val THOUSAND = 1000
+    private const val THOUSAND_LONG = 1000L
```

Comment:
Prefer semantic names instead. Both these constants refer to 1 second, so there should just be one constant named `ONE_SECOND`. There are extension methods available in `delay` to also accept `kotlin.time.Duration` types so you can just do `delay(ONE_SECOND.seconds)` instead.

Reply:
thanks for the tip, adjusted 👍